pipeline-runner/R/gem2s-7-upload_to_aws.R

```diff
@@ -128,8 +128,9 @@ meta_sets <- function(input, scdata, color_pool) {
   keys <- names(meta)
 
   # corresponding names stored in seurat object meta.data
-  seurat_keys <- scdata@misc$metadata_lookups[[keys]]
```

Comment:
Because if keys is a vector this doesn't work, right? 

That being said, the result of [] will be different than the result of [[]], won't this impact the type of seurat_keys

Reply:
`scdata@misc$metadata_lookups` is a named character vector so the only difference is that `seurat_keys` keeps the names with single bracket indexing. This doesn't have any consequences